bridge/engine/EngineMixer.cpp

```diff
@@ -2043,8 +2052,18 @@ void EngineMixer::processBarbellSctp(const uint64_t timestamp)
     for (IncomingPacketInfo packetInfo; _incomingBarbellSctp.pop(packetInfo);)
     {
         auto header = reinterpret_cast<webrtc::SctpStreamMessageHeader*>(packetInfo.packet()->get());
-        onBarbellUserMediaMap(packetInfo.transport()->getEndpointIdHash(),
-            reinterpret_cast<const char*>(header->data()));
+        auto message = reinterpret_cast<const char*>(header->data());
+        auto messageJson = utils::SimpleJson::create(message, strlen(message));
```

Comment:
Yes, I think it worth it. I did include it since I was worried, that since we can't guarantee that incoming json is indeed C-string, something like `utils::SimpleJson::create(const char*)` would tempt it to be used when it is not safe.